<launch>
  <rosparam file="$(find detector)/config/param_delay_landing.yaml"/>
  <node pkg="detector" type="simulation_detector" name="simulation_detector_node" output="screen" launch-prefix="gdb -ex run --args">
  <!--node pkg="detector" type="simulation_detector" name="simulation_detector_node" output="screen"-->
  </node>
</launch>
